objectivec/GPBArray.m

```diff
@@ -406,7 +406,7 @@ - (NSString *)description {
 }
 
 - (void)enumerateValuesWithBlock:(void (^)(int32_t value, NSUInteger idx, BOOL *stop))block {
-  [self enumerateValuesWithOptions:0 usingBlock:block];
+  [self enumerateValuesWithOptions:(NSEnumerationOptions)0 usingBlock:block];
```

Comment:
Oops, this needs to be updated in the template part of the file also.  That's what the travis failure is catching.